<launch>

  <param name="/pwm_roll" value="0" type="int"/>
  <param name="/pwm_pitch" value="0" type="int"/>
  <param name="/pwm_yaw" value="0" type="int" />

  <param name="/pwm_sway" value="0" type="int" />
  <param name="/pwm_surge" value="0" type="int" />
  <param name="/pwm_heave" value="0" type="int" />

  <param name="/current_task" value="marker_dropper_bottom" type="string" />

  <param name="/kill_signal" value="false" type="bool" />

  <param name="/reference_yaw" value="0" type="double" />

  <param name="/local_yaw" value="0" type="double" />

  <param name="/disable_imu" value="false" type="bool" />

  <param name="/use_reference_yaw" value="false" type="bool" />

  <param name="/use_local_yaw" value="false" type="bool" />

  <param name="/reference_depth" value="false" type="double" />

  <param name="/set_local_yaw" value="false" type="bool" />

  <param name="/set_reference_yaw" value="false" type="bool" />

</launch>
